lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include "PassDetail.h"
 
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
```

Comment:
nit: What do you need to use from Tensor?

Reply:
Removed (was experimenting with unrealized_conversion to use expand_shape and collapse_shape)